src/SourceBuild/content/Directory.Build.targets

```diff
@@ -10,6 +10,20 @@
     </ItemGroup>
   </Target>
 
+  <!-- Used by build.proj and repo-projects/package-source-build.proj -->
```

Comment:
I find that these kinds of comments easily become obsolete as changes are made in the sources without such comments being updated. This information can be fairly easily derived by doing a file search instead.

Reply:
Agreed - will update. Thanks!